tests/libcxx/host/host.cpp

```diff
@@ -5,8 +5,15 @@
 #include <openenclave/internal/calls.h>
 #include <openenclave/internal/error.h>
 #include <openenclave/internal/tests.h>
+#include <inttypes.h>
+#if defined(_MSC_VER)
+#include <windows.h>
+typedef DWORD pthread_t;
+#else
 #include <pthread.h>
```

Comment:
can we just use C++ `std::thread` instead of pthread? It is already cross-platform so we don't need to have ifdefs